scripts/v2/post.js

```diff
@@ -733,6 +734,15 @@ function decorateEmbeds() {
       type = 'twitter';
     }
 
+    if (firstLvl === 'slideshare' && isEmbedUrl($a.href, urlList[firstLvl])) {
+      embedHTML=`<div style="left: 0; width: 100%; height: 0; position: relative; padding-bottom: 61%;">
+      <iframe src="${url.href}" style="border: 0; top: 0; left: 0; position: absolute; width: 100%; height: 100%;" allowfullscreen="" scrolling="no" allow="encrypted-media; accelerometer; gyroscope; picture-in-picture" title="content from youtube" loading="lazy"></iframe>
```

Comment:
`title = "content from youtube"`, I assume this is incorrect?

Reply:
thanks! fixed.